docs/agents/sematext-agent/kubernetes/helm.md

```diff
@@ -0,0 +1,108 @@
+## Introduction
+
+This chart installs the Sematext Agent to all nodes in your cluster via a `DaemonSet` resource.
+
+## Prerequisites
+
+- Kubernetes 1.13+
+- You need to create [an Infra App in Sematext Cloud](https://apps.sematext.com/ui/monitoring-create/app/infra) to get your infra token.
+
+## Installation
+
+To install the chart for monitoring and shipping logs run the following command:
+
+```bash
+$ helm repo add sematext https://cdn.sematext.com/helm-charts/
+$ helm install st-agent \
```

Comment:
Shouldn't this have a logs app token then?

Reply:
With discovery, this is configured automatically, so the logs app token is not needed explicitly

Reply:
OK